### PR TITLE
Add ARGB support

### PR DESCRIPTION
**Pull request by alxarsnk**

### CODE REVIEW

**Comment by alxarsnk** [comment]
@thii check it please

File: Sources/SwiftHEXColors/SwiftHEXColors.swift [hunk 1]
```diff
@@ -117,4 +117,36 @@ public extension SWColor {
             return nil
         }
     }
+    
+    convenience init?(ARGBhex: Int) {
```
Comment (thii) [inline Sources/SwiftHEXColors/SwiftHEXColors.swift:121]:
```suggestion
    convenience init?(argbHex: Int) {
```

File: Sources/SwiftHEXColors/SwiftHEXColors.swift [hunk 2]
```diff
@@ -117,4 +117,36 @@ public extension SWColor {
...
+                      green: CGFloat( (hex & 0x0000FF00) >> 8 ) / 255.0,
+                      blue:  CGFloat( (hex & 0x000000FF) >> 0 ) / 255.0,
+                      alpha: CGFloat( (hex & 0xFF000000) >> 24 ) / 255.0)
+        } else {
+            self.init()
+            return nil
+        }
+    }
+    
+    convenience init?(ARGBhexString: String) {
```
Comment (thii) [inline Sources/SwiftHEXColors/SwiftHEXColors.swift:134]:
```suggestion
    convenience init?(argbHexString: String) {
```

**Review by thii — COMMENTED** [review]
Can you add a simple test?

**Comment by alxarsnk** [comment]
@thii fixed your issues and add couple of simple tests

**Review by thii — APPROVED** [review]
Thanks!

**Pull request merged**